docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6304,6 +6328,409 @@ field [@ProtoAny]. At the moment, there is not any mechanism to extend the
+      Wildcard doesn't apply to default_entry.
+    * `READ` : The Read RPC. All entries can be read. If no entities are
+      provided in request, then all entries are read, AKA wildcard read.
+      If `default_entry` is true, then the default entry is returned. Match
+      fields should not be set in this case.
+      Wildcard read does NOT return default entry.
+
+  * Modify-only tables: Entries exists from the time of initialization. Entries
+    cannot be inserted but only modified. Delete resets the entry to init time.
+    Read works as intended.
+    * `INSERT` : Not supported. Returns `NOT_SUPPORTED` error.
+    * `MODIFY` : Union field must be set since all data fields are under a
+      union. Modifies the union spec to new union or new field values if
+      same union.
+      If `default_entry` is true, then the default entry is modified. Match
+      fields and priority should not be set. Const default union rules apply.
+    * `DELETE` : Instead of deleting, this operation resets the entry with
+      default values for every field. The union field values are changed
+      but the union itself isn't changed.
+      Only Match fields must be set. `NOT_FOUND` is returned if
+      the entry isn't found. If no Match fields are given, then all entries
+      are reset (wildcard reset). If the table contained init entries, they
+      are restored to initial state with their original union and union field
+      values.
+      If `default_entry` is true, then the default entry is reset to initial
+      values. Const union rules apply.
+      Wildcard doesn't apply to default_entry.
+    * `READ` : The Read RPC. All entries can be read. If no entities are
+      provided in request, then all entries are read, AKA wildcard read.
+      If `default_entry` is true, then the default entry is returned. Match
+      fields should not be set in this case.
+      Wildcard read does NOT return default entry.
+
+  * Read-only tables: Entries can only be read. Write RPC doesn't work and
+    `NOT_SUPPORTED` is returned.
+    * `READ` : The Read RPC. All entries can be read. If no entities are
+      provided in request, then all entries are read, AKA wildcard read.
+      If `default_entry` is true, then the default entry is returned. Match
+      fields should not be set in this case.
+      Wildcard read does NOT return default entry.
+
+  * Reset-only tables :
+    * `INSERT` : Not supported. Returns `NOT_SUPPORTED` error
+    * `MODIFY` : Not supported. Returns `NOT_SUPPORTED` error
+    * `DELETE` : Instead of deleting, this operation resets the entry with
+      default values for every field.
+      Only Match fields must be set. `NOT_FOUND` is returned if
+      the entry isn't found. If no Match fields are given, then all entries
+      are reset (wildcard reset). If the table contained init entries, they
+      are restored to initial state with their original union and union field
+      values.
```

Comment:
Same comment for wildcard reset as for wildcard operations on other table types mentioned earlier.